### ISSUE TITLE
Add an "My posts" link to the stream menu

### ISSUE THREAD

**Issue by GergelySzekely** [noplanman/DiasPlus#17]
If it is possible, add a "My Posts" link to the stream menu that list only those posts that were posted by the user.

**Comment by noplanman** [comment]
Hi @GergelySzekely 😃 

Why don't you just click on the person's profile? That lists all posts by that person.
For your own profile, you can click the profile picture on the main stream page at the top, next to the post input. Or on the user menu on the top right, then "Profile".

Or do you mean something more specific?

**Comment by GergelySzekely** [comment]
No. I meant listing only the original post (excluding the reshared ones). 
If it were also possible to list other people's original post (excluding the reshared ones) that would be even better.

**Comment by noplanman** [comment]
Aaaah, now I got you 👍 

Will check it out!
